### ISSUE TITLE
Mismatch of HNT earned between Dashboard and Device Manager

### ISSUE THREAD

**Issue by bigdaveakers** [NebraLtd/dashboard-issues#10]
**Describe the bug**
The HNT earned in the last 24 hour period is mismatched between the Dashboard and the Device Manager. This may be related to issue #9  

**To Reproduce**
1. Examine Dashboard
2. Compare to Device Manager

**Expected behavior**
A match between Dashboard and Device Manager

**Screenshots**
Last period 0.33, current period x
![image](https://user-images.githubusercontent.com/3410330/119946829-f3b63d00-bf8e-11eb-8be2-af5c580a5dbb.png)
Showing stats only up to last period
![image](https://user-images.githubusercontent.com/3410330/119946974-1cd6cd80-bf8f-11eb-8ccc-f955a67768a7.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

**Comment by shawaj** [comment]
Interesting. I think we get all of these from the Helium API. Does it match with what is in the Helium explorer?

**Comment by vpetersson** [comment]
Yeah we do pull these directly from Helium's API. Thinking out loud, this might be related to timezone setting perhaps.

**Comment by shawaj** [comment]
I believe @bigdaveakers is in UK, FWIW

**Comment by bigdaveakers** [comment]
Yep I am in the UK, but it appears still to be out of sync - see screenshots from 10:49am UK

![image](https://user-images.githubusercontent.com/3410330/120065892-9d6afc00-c06b-11eb-941f-4c0e9d66999b.png)


![image](https://user-images.githubusercontent.com/3410330/120065906-aa87eb00-c06b-11eb-80a6-42013928495e.png)

**Comment by Nayonac1** [comment]
Possibly an API issue, as im seeing a similar issue on Sitebot when looking at one of my Miners (still waiting on nebra batch 1 delivery but will keep an eye on this once they arrive)  

Sitebot clearly saying this miner has made 17.9 
![image](https://user-images.githubusercontent.com/82951375/120321053-c2b17180-c2da-11eb-9d23-d9ba2d73be67.png)

From helium explorer (also checked back 48 hours and 7 days not enougt to make 17 
![image](https://user-images.githubusercontent.com/82951375/120321181-e4125d80-c2da-11eb-9ddb-6125430ec266.png)

Confirmed with a 3rd source (hotspotty)
![image](https://user-images.githubusercontent.com/82951375/120321381-2471db80-c2db-11eb-9f6a-ea8ba77788fb.png)

**Comment by vpetersson** [comment]
My guess would be that Helium's API uses UTC and they might have some other internal way of accessing this data.
